tests/ci/run_accp_dieharder.sh

```diff
@@ -3,5 +3,22 @@ set -exo pipefail
 # Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0
 
+# Testing non-FIPS is the default.
+testing_fips=false
+while [[ $# -gt 0 ]]; do
+    case ${1} in
+    --fips)
+      testing_fips=${2}
+      shift
```

Comment:
don't you need to call `shift` twice -- one for the `--fips` one for the `true`/`false`?

related nit: do you really need the `true`/`false` argument to the `--fips` flag? conventionally, boolean values are passed by mere presence of absence of a `--boolean-flag`. to simplify it a bit, you could do

```
testing_fips=false
while [[ $# -gt 0 ]]; do
    case ${1} in
    --fips)
      testing_fips=true
      shift
      ;;
    *)
      echo "${1} is not supported."
      exit 1
      ;;
    esac
    # Check next option -- key/value.
    shift
done

echo "Testing ACCP dieharder tests."
./gradlew dieharder
./gradlew -DFIPS=$testing_fips dieharder
```

Reply:
> don't you need to call shift twice -- one for the --fips one for the true/false?

The `shift` before the done is second `shift.

I think the `--boolean-flag` makes perfect sense. Adjusted it accordingly.

Reply:
ah, whoops. sorry i missed the `shift` on line 20.